styles/scss/_general.scss

```diff
@@ -0,0 +1,25 @@
+// * {border: 1px solid red;}
+
+h1, h2, h3, p {
+   margin: 0;
+   padding: 0;
+}
+
+img {
```

Comment:
cool if we add a bit more to the img styles in the set-up here? I usually add the below based on a pro-tip from Esther back in webdev days to make the imgs easier to work with!
img {
max-width: 100%;
height: auto;
display: block;
}

Reply:
yes! I was feeling that the img was missing some styles, thank you for pointing that out!

Reply:
No worries! Thanks for adding it in! 😄